examples/arkane/reactions/NH2+N2H3=NH+N2H4/N2H3.yml

```diff
@@ -0,0 +1,118 @@
+RMG_version: 2.3.0
+adjacency_list: 'multiplicity 2
+
+  1 N u0 p1 c0 {2,S} {3,S} {4,S}
+
+  2 N u1 p1 c0 {1,S} {5,S}
+
+  3 H u0 p0 c0 {1,S}
+
+  4 H u0 p0 c0 {1,S}
+
+  5 H u0 p0 c0 {2,S}
+
+'
+chemkin_thermo_string: "N2H3                    H   3N   2          G    10.000  3000.000
+  \ 478.04      1\n 2.42935350E+00 1.15088524E-02-6.01470204E-06 1.59974609E-09-1.70089224E-13
+  \   2\n 2.62991058E+04 1.15861374E+01 4.04880122E+00-4.38083729E-03 5.11833994E-05
+  \   3\n-8.84029667E-08 5.22512137E-11 2.61709988E+04 5.24801815E+00                   4\n"
```

Comment:
Do you have any idea why this string is printed differently from the adjlist?

Reply:
I think it's because the lines are too long, so YAML breaks them, then makes it a multiline string with `"..."`. Perhaps `\n`s aren't executed then?